src/dataDisplay/popover/popover.jsx

```diff
@@ -0,0 +1,140 @@
+import Classnames from 'classnames';
+import { get } from 'lodash';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+    UI_CLASS_NAME,
+    BEM_POPOVER,
+} from '../../global/constants';
+import makeStyles from '../../styles/makeStyles';
+import ToolTip from '../tooltip';
+
+const propTypes = {
+    /**
+     * The arrow can be enabled/disabled
+     */
+    arrow: PropTypes.bool,
+    /**
+     * Override or extend the styles applied to PersonPanel.
+     */
+    classes: PropTypes.shape({
+        root: PropTypes.string,
+    }),
+    /**
+     * Additional classes.
+     */
+    className: PropTypes.string,
+    /**
+     * Content
+     */
+    content: PropTypes.string,
+    /**
+     * We can pass either a string or jsx elements
+     */
+    children: PropTypes.shape({}).isRequired,
+    /**
+     * The `id` of the Popover.
+     */
+    id: PropTypes.string,
+    /**
+     * Popper container styles
+     */
+    popperStyles: PropTypes.shape({}),
```

Comment:
Not a blocker: I'm curious `popperStyles`. I'll figure it out some other day. For now, looks good.